config/environments/production.rb

```diff
@@ -55,7 +55,12 @@
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :mem_cache_store, ENV["MEMCACHEDCLOUD_SERVERS"].split(','), { username: ENV["MEMCACHEDCLOUD_USERNAME"],
+                                                                                     password: ENV["MEMCACHEDCLOUD_PASSWORD"],
+                                                                                     namespace: 'classroom',
+                                                                                     expires_in: 15.minutes,
```

Comment:
This seems too long, what do you think about 5 minutes? Or even less than that. Maybe 2?